Update tutorial to focus on deploying Node.js apps (#177)
IBM/zowe-cli-cics-deploy-plugin: docs/pages/cdp/cdp-Home.md

title: Zowe CLI CICS deploy plugin
tags: [getting_started]
keywords:
summary: "The Zowe command line interface (CLI) provides a simple and streamlined way to interact with IBM z/OS. The cics-deploy plugin extends the Zowe CLI to deploy applications developed on a workstation to IBM CICS Transaction Server for z/OS (CICS)."
sidebar: cdp_sidebar
permalink: index.html
folder: cdp
---

It provides a CICS deployment workflow for developers and CI/CD automation pipelines similar to that experienced when deploying applications to cloud environments.

Before you start, and especially if you are not familiar with the world of z/OS, Zowe and CICS, it's a good idea to read about some of the key concepts, which include [Zowe](cdp-zowe-and-cli), [Zowe CLI profiles](cdp-zowe-profiles) and [CICS bundles](cdp-cics-bundles). If you're keen to get going quickly, you can jump right in and follow along with our tutorial on [Deploying your first Node.js app](cdp-Deploying-your-first-nodejs-app).

> When everything has been [installed](cdp-Installation) and you've [created the Zowe CLI profiles](cdp-Create-Zowe-CLI-profiles), you can deploy an  application from your workstation to CICS in two steps:

Reviewer: ```suggestion
When everything is [installed](cdp-Installation) and you have [created the Zowe CLI profiles](cdp-Create-Zowe-CLI-profiles), you can deploy an application from your workstation to CICS in two steps:
```
